incubator/nodejs-express/image/project/Dockerfile

```diff
@@ -7,16 +7,23 @@ RUN apt-get update \
  && apt-get clean \
  && echo 'Finished installing dependencies'
 
+# Copying individual files/folders as buildah 1.9.0 does not honour .dockerignore
+COPY package*.json ./
```

Comment:
@neeraj-laad Is this COPY correct? Where are the package*.json files getting copied to?
Where this was moved from we had done a `WORKDIR /project` first so I presume the .json files were being copied into /project, but with it as we have now are they getting copied into the correct directory.

Reply:
Good catch..